plugins/spu2-x/src/Dma.cpp

```diff
@@ -420,12 +413,10 @@ void V_Core::DoDMAwrite(u16 *pMem, u32 size)
     }
     Regs.STATX &= ~0x80;
 //Regs.ATTR |= 0x30;
-#endif
 }
 
 s32 V_Core::NewDmaRead(u32 *data, u32 bytesLeft, u32 *bytesProcessed)
 {
-#ifdef ENABLE_NEW_IOPDMA_SPU2
```

Comment:
This is ifdef, not ifndef.

Actually, you could probably remove NewDmaRead, NewDmaWrite, and NewDmaInterrupt.

Reply:
I was worried I'd miss one of those, since it kept varying between #ifdef and #ifndef. And yeah, I should probably remove the functions for those three and the ones above as well. I'll look at it when I have a chance.